app/controllers/api/v1/keywords_controller.rb

```diff
@@ -3,10 +3,12 @@
 module Api
   module V1
     class KeywordsController < ApplicationController
+      include Pagy::Backend
       after_action :verify_policy_scoped, only: :index
+
       def index
-        keywords = policy_scope(Keyword)
-        render json: KeywordSerializer.new(keywords).serializable_hash.to_json
+        pagy, keywords = pagy(policy_scope(Keyword), pagination_params)
```

Comment:
Let's use the proper name for it, `pagy` is the name of library. 🎨 

```suggestion
        pagination, keywords = pagy(policy_scope(Keyword), pagination_params)
```

Reply:
solved in fe2c3fd